addons/hr_expense/models/hr_expense.py

```diff
@@ -268,6 +313,7 @@ def _compute_duplicate_expense_ids(self):
                 exp.duplicate_expense_ids = [(6, 0, ids)]
                 expenses = expenses - exp
 
+    # sgv todo replace with depends?
```

Comment:
I wouldn't do it for the moment. But you can leave your comment :)

Reply:
Agreed :)